### DISCUSSION TITLE
Add support to paste screenshots from clipboard

### DISCUSSION THREAD

**Discussion by mohitgangwani** [adityaoberai/Alt-Text-Generator#20] [Ideas]
Instead of using a saved file, if it's possible, allow the user to paste the screenshot from the clipboard.
I use a screenshot tool (Lightshot) so every time whenever I have to generate alt text, I have to save it. Kind of a hassle though. Will be bump in a UX imo.

**Comment by adityaoberai** [comment]
That's a great point! Will add this in :)

**Reply by mohitgangwani** [reply]
I don't know if this should be discussed in this discussion here, but you can also consider option of adding drag and drop functionality.

**Reply by adityaoberai** [reply]
> I don't know if this should be discussed in this discussion here, but you can also consider option of adding drag and drop functionality. 

You can drag an image onto the input field and it will upload it.

**Reply by mohitgangwani** [reply]
Oh alright, I guess then front end needs to be updated for this, I didn't have idea that drag and drop is supported.

**Comment by adityaoberai** [comment]
Forgot to mention that this was added in earlier

https://github.com/adityaoberai/Alt-Text-Generator/commit/d677623b6a276379a9272954a9fef5eef2a1e139
